Resources/Prototypes/_CM14/Entities/Structures/Xeno/xeno_doors.yml

```diff
@@ -2,6 +2,7 @@
   id: DoorXenoResin # TODO CM14 make them only buildable next to other resin walls
   parent: CMBaseXenoStructure
   name: resin door
+  description: A resin door that only sisters may pass.
```

Comment:
These descriptions are only supposed to be seen by xenos in cm13, I'll add a component to make this possible.
This is what a marine sees when examining a resin door:
![image](https://github.com/CM-14/CM-14/assets/10968691/14642605-0b89-4a78-92b5-0314768db5a4)